endorser/services/endorsement_processor.py

```diff
@@ -274,16 +153,35 @@ def _handle_unprocessable_endorse_event(
         bound_logger = logger.bind(body={"key": key})
         bound_logger.warning("Handling problematic endorsement event")
 
-        unprocessable_key = f"unprocessable:{key}"
+        unprocessable_key = f"unprocessable.{key}"
         error_message = f"Could not process: {event_json}. Error: {error}"
 
         bound_logger.info(
             "Saving record of problematic event at key: {}. Error: `{}`",
             unprocessable_key,
             error_message,
         )
-        self.redis_service.set(key=unprocessable_key, value=error_message)
-
-        bound_logger.info("Deleting original problematic event")
-        self.redis_service.delete_key(key=key)
+        await self.jetstream.publish(unprocessable_key, error_message.encode())
         bound_logger.info("Successfully handled unprocessable event.")
+
+    async def _subscribe(self) -> JetStreamContext.PullSubscription:
+        """
+        Subscribes to the NATS subject for endorsement events.
+        """
+        logger.info("Subscribing to NATS subject: cloudapi.aries.events.endorser.>")
+        try:
+            subscribe_kwargs = {
+                "durable": ENDORSER_DURABLE_CONSUMER,
+                "subject": f"{NATS_SUBJECT}.endorser.>",
```

Comment:
subject is logged (hardcoded) and then defined as f"{NATS_SUBJECT}.endorser.>"

We can also use `f"{NATS_SUBJECT}.endorser*"`, since > will capture extra groups, which we don't expect